cpp/oneapi/dal/algo/svm/test/batch.cpp

```diff
@@ -0,0 +1,607 @@
+}
+
+TEMPLATE_LIST_TEST_M(svm_batch_test,
+                     "svm can classify quadric separable surface with rbf kernel",
+                     "[svm][integration][batch]",
+                     svm_types) {
+    SKIP_IF(this->not_available_on_device());
+
+    using Float = std::tuple_element_t<0, TestType>;
+    using Method = std::tuple_element_t<1, TestType>;
+    using KernelTypeRBF = typename rbf::descriptor<Float, rbf::method::dense>;
+
+    constexpr std::int64_t row_count_train = 12;
+    constexpr std::int64_t column_count = 2;
+    constexpr std::int64_t element_count_train = row_count_train * column_count;
+
+    constexpr std::array<Float, element_count_train> x_data = { -2.0, 0.0, -2.0, -1.0, -2.0, 1.0,
+                                                                2.0,  0.0, 2.0,  -1.0, 2.0,  1.0,
+                                                                -1.0, 0.0, -1.0, -0.5, -1.0, 0.5,
+                                                                1.0,  0.5, 1.0,  -0.5, 1.0,  0.5 };
+    const auto x = homogen_table::wrap(x_data.data(), row_count_train, column_count);
+
+    constexpr std::array<Float, row_count_train> y_data = { -1.0, -1.0, -1.0, -1.0, -1.0, -1.0,
+                                                            1.0,  1.0,  1.0,  1.0,  1.0,  1.0 };
+    const auto y = homogen_table::wrap(y_data.data(), row_count_train, 1);
+
+    const double sigma = 1.0;
+    const double c = 1.0;
+
+    const auto kernel_desc = KernelTypeRBF{}.set_sigma(sigma);
+    const auto svm_desc =
+        svm::descriptor<Float, Method, svm::task::classification, KernelTypeRBF>{}.set_c(c);
+
+    constexpr std::int64_t support_vector_count = 12;
+
+    constexpr std::array<Float, support_vector_count> support_indices_data = { 0, 1, 2, 3, 4,  5,
+                                                                               6, 7, 8, 9, 10, 11 };
+    const auto support_indices =
+        homogen_table::wrap(support_indices_data.data(), support_vector_count, 1);
+
+    this->check_rbf_kernel(x,
+                           y,
+                           svm_desc,
+                           support_vector_count,
+                           support_indices,
+                           homogen_table{},
+                           y);
+}
+
+TEMPLATE_LIST_TEST_M(svm_batch_test,
+                     "svm can classify any two labels",
+                     "[svm][integration][batch]",
+                     svm_types) {
+    SKIP_IF(this->not_available_on_device());
+
+    using Float = std::tuple_element_t<0, TestType>;
+    using Method = std::tuple_element_t<1, TestType>;
+    using KernelTypeLinear = typename linear::descriptor<Float, linear::method::dense>;
+
+    constexpr std::int64_t row_count_train = 6;
+    constexpr std::int64_t column_count = 2;
+    constexpr std::int64_t element_count_train = row_count_train * column_count;
+
+    using LabelsPair = std::pair<std::array<Float, row_count_train>, std::array<Float, 2>>;
```

Comment:
```suggestion
    using labels_pair_t = std::pair<std::array<Float, row_count_train>, std::array<Float, 2>>;
```

Reply:
Also suggest to move type aliases to the beginning of method where others are defined

Reply:
In this case I use row_count_train. Should I remove all constexpr (row_count / column_count) and use only numerical values?